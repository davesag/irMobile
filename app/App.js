/**
 * @format
 */
import React from 'react'
import { Provider } from 'react-redux'
import { Text } from 'react-native'
import { Provider as ThemeProvider } from 'react-native-paper'
import FlashMessage from 'react-native-flash-message'

import AppMain from './AppMain'

import store from './services/store'
import theme from './theme'

// TODO: work out a better way to approach this.
Text.allowFontScaling = false

const App = () => (
  <Provider store={store}>
    <ThemeProvider theme={theme}>
      <AppMain />
      <FlashMessage position="bottom" />
    </ThemeProvider>
  </Provider>
)

export default App
