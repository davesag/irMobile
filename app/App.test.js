import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import App from './App'

let tree

describe('rendering the component', () => {
  beforeAll(() => {
    tree = renderer.create(<App />)
  })

  it('rendered correctly ', () => {
    expect(tree).toMatchSnapshot()
  })
})
