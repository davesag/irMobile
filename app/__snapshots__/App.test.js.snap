// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering the component rendered correctly  1`] = `null`;
